File: 1223/9x9.rb
#!/usr/bin/ruby

ary = (1..9).to_a

for i in ary
	for j in ary
		ans = i*j
		if ans <= 9 then
			print "  ", i*j
		else
			print " ", i*j
		end
	end
	print "\n"
end

File: 0120/9x9.py
#!/usr/bin/env python3

array = range(1, 10)

for i in array:
    for j in array:
        ans = i*j
        if ans <= 9:
            print(" ", i*j, end="")
        else:
            print("", i*j, end="")
    print("")
